src/main/java/Command.java

```diff
@@ -0,0 +1,138 @@
+public class Command {
+
+    private String[] command;
+
+
+    /**
+     * Creates a Command object
+     *
+     * @param input A string representation of user's command.
+     */
+    public Command(String input) {
+        this.command = input.split("\\s", 2);
```

Comment:
Parsing of input can be move to Parser class!

Reply:
Agreed, this would help your code to better achieve the Single responsibility Principle.